app/views/posts/_the_user.html.erb

```diff
@@ -0,0 +1,7 @@
+  <div class='users'>
+    <div class='users_pic'></div>
+    <div class='users_name'>
+      <h2>User <%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_count %></p>
+    </div>
+  </div>
```

Comment:
- [ ] Kindly, fix this issue, your project is not running because of this issue.

![pots](https://user-images.githubusercontent.com/55635977/144512602-db0477f2-cd1b-4018-9059-30182eaac3cd.png)

Reply:
- [ ] My project is running perfectly find I suggest you check your side or give me a hint instead. 

![Screen Shot 2021-12-02 at 16 27 12](https://user-images.githubusercontent.com/67211919/144514159-a9cf20e1-fc3c-4d08-ba21-420adf0c212c.png)